src/reducetensor.cpp

```diff
@@ -59,6 +60,15 @@ enum ReductionMethod_t
 
 namespace detail {
 
+static bool IsDynamicReductionEnabled()
```

Comment:
[Recommendation] Buttery butter ;) Either static or namespace.
```suggestion
bool IsDynamicReductionEnabled()
```

Reply:
[Recommendation] Use unnamed namespace.